Main.java

```diff
@@ -0,0 +1,38 @@
+package LAB7.LAB7;
```

Comment:
package names are lowe letters and NEVER should duplicate the name two times without strong need

Reply:
reworked and uploaded to another repository 
https://github.com/bohdanshcherba/javalab/pull/1